test/Tizen.NUI.WindowSystem.Samples/Tizen.NUI.WindowSystem.Samples.cs

```diff
@@ -0,0 +1,322 @@
+﻿using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Components;
+using Tizen.NUI.WindowSystem;
+
+namespace Tizen.NUI.WindowSystem.Samples
+{
+    class Program : NUIApplication
+    {
+        Shell.TizenShell tzShell;
+        Shell.QuickPanelClient qpClient;
+        Button BtnClient;        
+        TextLabel textClientVisible;
+        TextLabel textClientScrollable;
+        TextLabel textClientOrientation;
+
+        Shell.QuickPanelService qpService;
+        Shell.TizenRegion qpRegion;
+        Button BtnService;
+
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+            Initialize();
+        }
+
+        void Initialize()
+        {
+            Window.Instance.KeyEvent += OnKeyEvent;
+            Window.Instance.BackgroundColor = Color.White;
+
+            BtnClient = new Button()
+            {
+                Text = "QuickPanelClient",
+                Size = new Size(400, 100),
+                Position2D = new Position2D(100, 200),
```

Comment:
Please use Position(100,200) instead of Position2D().

Reply:
I changed all Position2D to Position in this code